### PR TITLE
[bitnami/*] Adapt values.yaml of phpMyAdmin, PostgreSQL and PostgreSQL HA charts

### PR DESCRIPTION
**Pull request by pablogalegoc**
**Description of the change**

Add metadata to `values.yaml` in order to be used by [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) to automatically generate the "Parameters" section of the `README.md`.

General rules followed:
- If `values.yaml` doesn't have a description for a key in a comment → Use `README.md` description
- If `values.yaml` does have a description for a key in a comment but the `README.md` description provides more info → Use `README.md` description
- Else → Leave the `values.yaml` description

**Benefits**

Less manual work and consistency between `values.yaml` and `README.md`.

**Additional information**

Commented keys are not picked by readme-generator so some had to be uncommented in order to appear in the README, however none of them affect the default installation of the chart, verified comparing the output before and after the changes of
```bash
helm install chart . -f values.yaml --dry-run --debug
```

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/phpmyadmin/values.yaml [hunk 1]
```diff
@@ -20,112 +58,102 @@ image:
...
+## @param extraEnvVarsCM Name of a existing ConfigMap containing extra env vars
 ##
-clusterDomain: cluster.local
-
-## Extra objects to deploy (value evaluated as a template)
+extraEnvVarsCM:
+## @param extraEnvVarsSecret Name of a existing Secret containing extra env vars
 ##
-extraDeploy: []
+extraEnvVarsSecret:
```
Comment (alvneiayu) [inline bitnami/phpmyadmin/values.yaml:84]: I was confused with this change but I saw that you are reordering keys

File: bitnami/postgresql-ha/values.yaml [hunk 2]
```diff
@@ -310,153 +276,166 @@ postgresql:
...
-
-  ## Store Repmgr in a custom path
-  # repmgrPassfilePath:
-
-  ## Upgrade repmgr extension in the database
+  ##
+  postgresPassword:
+  ## @param postgresql.usePasswordFile Set to `true` to mount PostgreSQL secret as a file instead of passing environment variable
+  ##
+  usePasswordFile:
```
Comment (alvneiayu) [inline bitnami/postgresql-ha/values.yaml:340]: Attention with this, this key was commented and right now no. Maybe this is going to change the default behavior of the chart. Did you test this change?

File: bitnami/postgresql-ha/values.yaml [hunk 3]
```diff
@@ -310,153 +276,166 @@ postgresql:
...
+  ## @param postgresql.repmgrUsePassfile Set to `true` to configure repmgrl to use `passfile` instead of `password` vars*:*:*:username:password" and use it to configure Repmgr instead of using password (Requires Postgresql 10+, otherwise ignored)
+  ## ref: https://repmgr.org/docs/current/configuration-password-management.html
+  ##      https://www.postgresql.org/docs/current/libpq-pgpass.html
+  ##
+  repmgrUsePassfile:
```
Comment (alvneiayu) [inline bitnami/postgresql-ha/values.yaml:345]: same here

File: bitnami/postgresql-ha/values.yaml [hunk 4]
```diff
@@ -310,153 +276,166 @@ postgresql:
...
   ##
   username: postgres
-  # password:
-  # database:
-
-  ## PostgreSQL password using existing secret
-  # existingSecret: secret
-
-  ## PostgreSQL admin password (used when `postgresql.username` is not `postgres`)
+  password:
```
Comment (alvneiayu) [inline bitnami/postgresql-ha/values.yaml:329]: this line was commented and right now no

**Review by alvneiayu — CHANGES_REQUESTED** [review]
Hi @pablogalegoc 

First of all, really hard and good job. I saw that you are including keys that by default was commented. Did you test that the default chart behavior is working as expected?

File: bitnami/postgresql-ha/values.yaml (hunk 2 above)
Comment (pablogalegoc) [reply bitnami/postgresql-ha/values.yaml:340]: Verified that the manifests generated by the default values are not affected by diffing:
```bash
git checkout master values.yaml && helm install postgresql-ha . -f values.yaml --dry-run --debug > pre.yaml
```
```bash
git checkout HEAD values.yaml && helm install postgresql-ha . -f values.yaml --dry-run --debug > post.yaml
```

**Review by alvneiayu — APPROVED** [review]
LGTM

**Pull request merged**